src/plugins/data/public/query/state_sync/connect_to_query_state.ts

```diff
@@ -41,27 +40,28 @@ import { QueryState, QueryStateChange } from './types';
 import { FilterStateStore, COMPARE_ALL_OPTIONS, compareFilters } from '../../../common';
 import { validateTimeRange } from '../timefilter';
 
+export interface ISyncConfig {
+  filters: FilterStateStore;
+  query: boolean;
+  dataSet?: boolean;
```

Comment:
```suggestion
  dataset?: boolean;
```

Reply:
quick fix up in an old typo while we are in a neighbor

Reply:
hmm actuall when i think about it we no dont have a dataset outside of the query object so i think actually we could remove this field in the interface. 

wdyt? do you see it being used?